src/main/java/frc/robot/OI.java

```diff
@@ -49,7 +67,13 @@ public static JoystickButton getZeroButton(){
     }
 
     public static final class Operator{
+        private static final Joystick kJoystick = new Joystick(OI.kOperatorJoystickPort);
 
+        private static final int kBuddyBalanceToggleID = 0; //B Button, toggle buddy balance
+
+        public static JoystickButton ToggleBuddyBalanceButton(){
```

Comment:
Method names should be in camelCase, getter methods should start with get (getToggleBuddyBalanceButton())